behave/features/steps/specialist_provider.py

```diff
@@ -26,19 +28,36 @@ def step_impl(context):
     compare_client_details_with_backend(context, case_id, client_section)
 
 
-@step(u'I select a case from the dashboard')
+@step(u'I select a case to accept from the dashboard')
 def step_select_special_provider_case(context):
     case_reference = CLA_SPECIALIST_CASE_TO_ACCEPT
+    check_only_unaccepted_cases = True
+    select_a_case(context, case_reference, check_only_unaccepted_cases)
+
+
+@step(u'I select a case to reject from the dashboard')
+def step_impl(context):
+    case_reference = CLA_SPECIALIST_CASE_TO_REJECT
+    # Case doesn't need to accepted, can reject any case.
+    check_only_unaccepted_cases = False
+    select_a_case(context, case_reference, check_only_unaccepted_cases)
+
+
+def select_a_case(context, case_reference, check_only_unaccepted_cases):
 
     table = context.helperfunc.driver().find_element_by_css_selector(".ListTable")
+    unaccepted_check = "unaccepted" if check_only_unaccepted_cases else "accepted"
```

Comment:
this should be an empty string if not `check_only_unaccepted_cases` as it can be any case. 
Error messages are:
`Could not find case {case_reference} on the dashboard`
`Could not find unaccepted case {case_reference} on the dashboard`